.storybook/foundations/drop-shadow/index.css

```diff
@@ -0,0 +1,66 @@
+/*!
+Copyright 2024 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+@import "../../../components/commons/dropshadow.css";
```

Comment:
I'm still a little fuzzy on how this should work, but I believe once the CSS has been committed/released, this should change to `@import "@spectrum-css/commons/dropshadow.css";` and `.storybook/package.json` should have `@spectrum-css/commons` with the latest version as a devDependency, does that sound correct?

If that's the case, I'm thinking the easiest way to do this would be to make this commit a separate PR.

Reply:
This sounds correct to me based on how we're using commons/basebutton for example 👍